temporal_capi/src/time_zone.rs

```diff
@@ -33,6 +33,12 @@ pub mod ffi {
                 .map_err(Into::into)
         }
 
+        pub fn clone(&self) -> Box<Self> {
+            Box::new(Self(self.0.clone()))
+        }
+        pub fn utc() -> Box<Self> {
+            Box::new(Self(temporal_rs::TimeZone::IanaIdentifier("UTC".into())))
```

Comment:
question: any general thoughts on how this may update with any future changes around #330.

I'm mostly concerned about how changes to `TimeZone` may affect this in the future.